projects/01-pair-matching/Mahya-Behboudi/script.js

```diff
@@ -0,0 +1,139 @@
+// define variables 
+    const boardContainer = document.querySelector('.board-container')
+    const board = document.querySelector('.board')
+    const win = document.querySelector('.win')
+	const state ={flippedCards: 0, totalFlips:0,loop:null}
+
+// randomimize the number of cards 
+function randommize (a) {
+    const array = [...a]
+    for (let i = array.length - 1; i > 0; i--) {
+        const random = Math.floor(Math.random() * (i + 1))
+        const currentArray = array[i]
+        array[i] = array[random]
+        array[random] = currentArray
+    }
+
+    return array;
+}
+
+// pick the random of 
+function pickRandom(a, items)  {
+    const array = [...a]
+    const randomPicks = []
+
+    for (let i = 0; i < items; i++) {
+        const randomIndex = Math.floor(Math.random() * array.length)
+        
+        randomPicks.push(array[randomIndex])
+        array.splice(randomIndex, 1)
+    }
+
+    return randomPicks
+}
+
+
+//fetch API 
+async function applyNumber() {
+	// the default of api 
+	let abortApi = new AbortController();
+	const signal = abortApi.signal;
+	let getApiTimer = setTimeout(() => abortApi.abort(),3000);
+	let dimension = await fetch('http://192.168.43.20:8000/dimension',signal)
+		.then(response => response.json())
+		.then(response => {return response.dimension})
+
+//if the response is faild the dimension is 8
+		.catch(e => {
+			return 8})
+		clearTimeout(getApiTimer);
+		return dimension
+}
+//applyNumber();
```

Comment:
Please remove any redundant comments (like this one) from all your files.

Reply:
sure 💯